admin_server/routes/auth.js

```diff
@@ -0,0 +1,146 @@
+const router = require('express').Router();
+const axios = require('axios');
+const jwt = require('jsonwebtoken');
+
+const MSG91_URL = "https://control.msg91.com/api/";
+
+//----------Send OTP--------------
+router.post('/sendotp', async (req, res) => {
+  
+  try {
+
+    const params = {
+      country: req.body.country_code,
+      mobile: req.body.mobile_no,
+      authkey: process.env.MSG91_API_KEY,
+      sender: "HLOCHN",
+      message: "Your Hello Chennai app OTP is ##OTP##"
+    };
+    
+    const { data } = await axios.post(MSG91_URL+"sendotp.php",null, { params });
+
+    if(data.type === 'error'){
+
+      res.status(400).send({
+        message: ""+data.message,
+        status: "failed"
+      });
+
+    } else {
+      
+      res.status(200).send({
+        message: "sms sent",
+        status: "success"
+      });
```

Comment:
Think the status codes are self explanatory. do we need a seperate status message be sent back in the response ?

Reply:
for the sake of consistency and ease of development, i would like to have a consistent response message for every call. In any case even in regards to failure when implementing the api, the developer can expect a consistent response.